demo_event.c

```diff
@@ -315,6 +315,11 @@ int main(int argc, char *argv[])
 	if (signal(SIGINT, sig_handler) == SIG_ERR)
 		return -1;
 
+#ifdef NEED_GET_DATA
+	fprintf(stdout, "%s: UBLK_F_NEED_GET_DATA\n", __func__);		
```

Comment:
pass the flag via command line should be more flexible instead of config options.

Also please add one command line into ublksrv_tgt.c for using this feature, then add at least one
test case under tests/ for covering this new feature. With this way, it could be very easy to verify if
the feature can work well.

Reply:
Yes, I should add sth like: